src/NinjaTools.FluentMockServer.API/Configuration/ConfigFile.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics;
+using JetBrains.Annotations;
+using Newtonsoft.Json;
+using NinjaTools.FluentMockServer.API.Models;
+using YamlDotNet.Serialization;
+
+namespace NinjaTools.FluentMockServer.API.Configuration
+{
+    [DebuggerDisplay("{DebuggerDisplay(), nq}")]
+    public class ConfigFile : IConfigFile
+    {
+        [JsonConstructor]
+        public ConfigFile([NotNull] string path, [NotNull] params Setup[] setups)
+        {
+            Path = path;
+
+            var fileExtension = System.IO.Path.GetExtension(Path).TrimStart('.').ToLower();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4765163)